src/perf_capabilities.ml

```diff
@@ -93,6 +95,7 @@ let supports_kcore = kernel_version_at_least ~major:5 ~minor:5
 
 (* Added in kernel commit ce7b0e4, which made it into 5.4. *)
 let supports_snapshot_on_exit = kernel_version_at_least ~major:5 ~minor:4
+let supports_dlfilter = kernel_version_at_least ~major:5 ~minor:14
```

Comment:
Could you add the commit this was added in here, as a comment?

Reply:
Added